corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -104,9 +104,8 @@ hqDefine('analytix/js/hubspot', [
                             // the Schedule Once Form was submitted on our side.
                             // The style attribute changes when the form is successfully
                             // submitted.
-                            /* eslint-disable no-unblessed-new */
                             var lastKnownHeight = 0,
-                                observer = new MutationObserver(function (mutations) {
+                                observer = new MutationObserver(function (mutations) {  // eslint-disable-line eslint-dimagi/no-unblessed-new
```

Comment:
As mentioned in other comment, it's fine to add `MutationObserver` to the "blessed" list, but I'm curious, did the other style of disabling not work?

Reply:
yea the other style of disabling didn't work

Reply:
thanks, good to know